backbone/src/main/java/org/researchstack/backbone/ui/task/TaskViewModel.kt

```diff
@@ -51,6 +51,7 @@ internal class TaskViewModel(val context: Application, intent: Intent) : Android
     val moveReviewStep = MutableLiveData<StepNavigationEvent>()
     val showEditDialog = MutableLiveData<Boolean>()
     val updateCancelEditInLayout = MutableLiveData<Boolean>()
+    val stepInBackMode = MutableLiveData<Boolean>()
```

Comment:
how about renaming this? the current name doesn't say much to me, what is the BackMode? Are there more than one BackMode? Two back modes? True and false? what do they mean?